### PR TITLE
Display all graphs for one process

### PR DESCRIPTION
**Pull request by mbrauninger**

### CODE REVIEW

File: backend/client_sim.js [hunk 1]
```diff
@@ -48,7 +48,7 @@ const HOST = "localhost";
 const PORT = 49160;
 // The simulated client's Unique ID can be generated from the process' PID
 const UID = process.pid;
-const sendFreq = 1;
+const sendFreq = 10;
```
Comment (sdub18) [inline backend/client_sim.js:51]: Did bringing down the send frequency improve performance?

**Review by sdub18 — APPROVED** [review]
LGTM! No issues here, I think eventually we should sweep the project for any console.log statement we don't need, as this definitely bogs down the system. But otherwise this is great!

**Comment by andrewtran10** [comment]
LGTM👍
Remove code for selecting specific chart type

**Review by andrewtran10 — APPROVED** [review]
Looks good to me 👍 
Remove old code selecting specific chart type

**Pull request merged**